doc/configuration/introduction.md

```diff
@@ -1,25 +1,40 @@
 This chapter covers the node documentation, necessary to have a working system. It covers
 the network, logging and storage parameters.
 
-The node configuration uses the [YAML](https://en.wikipedia.org/wiki/YAML) format.
+## Node Configuration
+This is an common example of a Jormungandr node configuration file typically named `node-config.yaml`, however your's will vary depending on your needs.  Addtionally, this configuration has been tested on Jormungandr 0.7.1 and may change with newer versions.  It's important to keep in mind that the trusted_peers portion of this configuration will be different for each Cardano blockchain network.  If you're trying to connect this node to a specific network, you need to know it's genesis block hash, and it's associated list of trusted peers.
 
-This is an example of a configuration file:
+Example Configuration - 1:
 
 ```YAML
-storage: "/tmp/storage"
+---
 log:
-  level: debug
-  format: json
+  - output: stderr
+    level:  info
+    format: plain
 p2p:
-  trusted_peers:
-    - address: "/ip4/104.24.28.11/tcp/8299"
-      id: 0ccc678e5c41fcffc7398fc5cc9c4e08ba88934fe6565305
-    - address: "/ip4/104.24.29.11/tcp/8299"
-      id: 328c71454e1ecdf88fc5e3763c74997e117f0dd84ef6eddf
-  public_address: "/ip4/127.0.0.1/tcp/8080"
-  public_id: ad24537cb009bedaebae3d247fecee9e14c57fe942e9bb0d
+  public_address: "/ip4/40.90.149.161/tcp/3200" # This should match your public IP address.     
   topics_of_interest:
-    messages: low
-    blocks: normal
-  allow_private_addresses: false
+    blocks: normal #Default is normal - high for stakepool
+    messages: low   #Default is low - high for stakepool
```

Comment:
better removing them instead, they are already defaulted, it makes the default configuration example easier to read